packages/xo-server/src/utils.js

```diff
@@ -102,6 +102,9 @@ export const parseXml = (function () {
   const opts = {
     attributeNamePrefix: '',
     ignoreAttributes: false,
+    // Don't parse values to avoid trim error
```

Comment:
We don't want to parse the values because that's what the previous implementation used to do, this comment does not make sense for the current code.

```suggestion
```